### ISSUE TITLE
[🧊] [Next.js] Google AdSense 광고 적용 및 이해하기

### ISSUE THREAD

**Issue by YuHyeonWook** [1ilsang/dev#379]
안녕하세요. 구글 애드센스 심사 과정에 대해 궁금해서 이슈 남깁니다!
질문. 구글 애드센스 심사를 통과하려면 뉴스, 블로그 프로젝트이여야만 심사가 통과 되나요? 
현재 사이드 프로젝트로  구상했던 것은  광고 시청 후 응모권을 획득하여 상품을 응모하는 것인데 블로그 형식과는 다릅니다...

**Comment by 1ilsang** [comment]
> 질문. 구글 애드센스 심사를 통과하려면 뉴스, 블로그 프로젝트이여야만 심사가 통과 되나요?

아니요, [구글 게시자 정책](https://support.google.com/publisherpolicies/answer/10502938)에 어긋나지 않는다면 어떤 사이트든 상관없습니다.

정상적인 사이트라면 큰 이상 없이 통과하실 것 같아요. 😄
